packages/flutter/test/rendering/editable_test.dart

```diff
@@ -368,6 +376,38 @@ void main() {
     expect(editable, paintsExactlyCountTimes(#drawRect, 1));
   }, skip: isBrowser);
 
+  testWidgets('ignore key event from web platform', (WidgetTester tester) async {
```

Comment:
I am surprised that uses `TestWidgetsFlutterBinding.ensureInitialized();` still requires testWidget. what was the issue you were facing? If this is necessary, this test should be put in editable_text_test.dart

Reply:
Without testWidgets and pumpWidget, I could't use SystemBinding. :(
I will paste error message and sample code I tried because this can be user error(I'm newbie on Flutter).

I don't like this code which uses EditableText here. It's good if I can implement without testWidgets and pumpWidget function. and in other case, I will move this test code to editable_text_test.dart as you said.

Thank you.

Reply:
> I am surprised that uses `TestWidgetsFlutterBinding.ensureInitialized();` still requires testWidget. what was the issue you were facing? If this is necessary, this test should be put in editable_text_test.dart

Sorry today I succeed to make test code without testWidgets and pumpWidget functions as you said. maybe I was a little bit confused about test framework. :(
I updated my pull-request. Thank you for your comments. :)